source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,28 +7,42 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the host and  {engine-name} stop responding, and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
```

Comment:
@emarcusRH 
I spotted something that needs to be checked. I looked at the solutions article for manually renewing the host certificate. It is nearly the same procedure for enrolling the host certificates. Please ask an SME whether the user needs to manually renew the certificate AND to enroll the certificate. 

If this is the case (and it looks like it may be), I think we need to add the steps from the article to the procedure instead of providing a link. It is too easy for a user to miss a link in a warning callout.

If you look at the notice in the article, you can see that it's confusing.
![image](https://user-images.githubusercontent.com/31725355/168987016-33dd7731-57f8-4b78-ae14-afd381158001.png)

Reply:
Ack. That is my understanding as well. @sandrobonazzola or @michalskrivanek can you please check this and confirm or correct accordingly?

Reply:
@emarcusRH 

So you need to remove the link to article 3532921 from the warning. It is no longer relevant. These are the sentences to take out:

"If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates."

Reply:
@mtessun , @sandrobonazzola , @michalskrivanek 
I see that the KCS article :[https://access.redhat.com/solutions/3532921[How](https://access.redhat.com/solutions/3532921  How to manually renew RHV host SSL certificate if expired?
Was updated today - is this still relevant?

Reply:
@emarcusRH 
Nijin Ashok added this 10 hours ago:
![image](https://user-images.githubusercontent.com/31725355/169075981-fdc760df-6174-4891-a656-ae04e66301f4.png)
You can see the changes if you click the pencil and then click "Version timeline".

Reply:
> @emarcusRH
> 
> So you need to remove the link to article 3532921 from the warning. It is no longer relevant. These are the sentences to take out:
> 
> "If they do expire, you can manually renew them by following the instructions in link:[https://access.redhat.com/solutions/3532921[How](https://access.redhat.com/solutions/3532921%5BHow) to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates."

@emarcusRH This still needs to be removed. Martin Tessun approved the change.